x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/details/query.host_details.dsl.ts

```diff
@@ -58,6 +58,16 @@ export const buildHostDetailsQuery = ({
         },
       },
       query: { bool: { filter } },
+      _source: false,
+      fields: [
```

Comment:
Would like to know when there was no ```_source``` in previous query, how do we decide the `fields` here?

Reply:
I went through all queries and query results we have, and for this one we had _source in the aggregations results